docs/docs/external/getting-started/enabling-weaver-network/fabric.md

```diff
@@ -353,6 +558,112 @@ To start the driver, navigate to the folder containing the above files and run t
 docker-compose up -d
 ```
 
+#### Launch IIN Agents
+
```

Comment:
Do we need all of these instructions below just to start IIN Agents? This looks like a dump of the README, and quite intimidating. On this page, we should give as many shortcuts as possible (pull this image, configure env variables as follows, etc.), and then link to the README for details.

Reply:
It is similar to what you are suggesting (its same as what's in driver section), it is long because there are 3 config files required and then set of environment variables.

Reply:
I think it would be good to have it here, rather than pointing them to README, so that docs can standalone be enough to explain whatever is required.

Reply:
Can we provide instructions then in a less dry manner? Start with, to launch a single IIN agent, pull container image XYZ, edit the following configurations files in these ways, etc. Then say "To launch an additional IIN Agent corresponding to a different network member (organization in Fabric, node in Corda, etc.), repeat the above steps but change these specific things...". Something like this.

Try to edit this section a little bit along these lines. I can reorganize content in a later PR.